src/frontend/pages/Products/components/FiltersListing.jsx

```diff
@@ -0,0 +1,184 @@
+import React from "react";
+import { useProducts } from "../../../contexts/products-context";
+
+function Checkbox({ checked, label, changeType }) {
+  const { dispatch } = useProducts();
+
+  return (
+    <div className="pb-05">
+      <input
+        type="checkbox"
+        id={label}
+        value={label}
+        checked={checked}
+        onChange={() => dispatch({ type: changeType })}
+      />
+      <label className="p-1" htmlFor="scales">
```

Comment:
Either you wrap the input element on label for give name property on input so that when we click on text also , checkbox should be ticked.